cli/src/commands/mod.rs

```diff
@@ -485,6 +485,9 @@ struct DuplicateArgs {
 /// Abandon a revision, rebasing descendants onto its parent(s). The behavior is
 /// similar to `jj restore --changes-in`; the difference is that `jj abandon`
 /// gives you a new change, while `jj restore` updates the existing change.
+///
+/// Note that if you abandon the working-copy commit, a new empty working-copy commit will be created.
+/// If you want to remove the empty commit from the top of history, use jj unstash.
```

Comment:
You mean `unsquash` here, yes? I'm not quite sure how I feel about suggesting `unsquash` here (which keeps `@` and not `@-`), why not suggest `edit @-` here as well?

Also, the surrounding text seems to be quoting commands in backticks -- something to consider, for consistency.